Sources/BraveWallet/Crypto/CryptoView.swift

```diff
@@ -313,6 +313,7 @@ private struct CryptoContainerView<DismissContent: ToolbarContent>: View {
               toolbarDismissContent: toolbarDismissContent,
               onDismiss: {
                 cryptoStore.isPresentingPendingRequest = false
+                cryptoStore.prepare()
```

Comment:
Is this change needed? `prepare()` gets called in `BraveWalletTxServiceObserver`, this would add the case where user manually dismisses the pending request which might not be intended

Reply:
it's need since the `pendingRequest` which was `.transaction([])` need to be set to `nil`